tests/zero-downtime-scaling/results/1.2.0/1.2.0.md

```diff
@@ -0,0 +1,578 @@
+# Results
+
+## Versions
+
+Kubernetes:
+
+```text
+Server Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.8-gke.1067004", GitCommit:"6f460c12ad45abb234c18ec4f0ea335a1203c415", GitTreeState:"clean", BuildDate:"2024-01-04T22:48:32Z", GoVersion:"go1.20.11 X:boringcrypto", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+NGF:
+
+```text
+"version":"edge"
+"commit":"5b9f37c6556388e28d2cd168810b728cca4c0e46"
+"date":"2024-03-19T20:53:54Z"
+```
+
+with NGINX:
+
+```text
+nginx version: nginx/1.25.4
+Built by gcc 12.2.1 20220924 (Alpine 12.2.1_git20220924-r10)
+OS: Linux 5.15.133+
+```
+
+## Summary
+
+Results are similar with 1.1.0 test results. Noticed some spikes in NGF log errors when downgrading the number of replicas from 25 to 1 related to telemetry. Further investigating might be needed.
```

Comment:
For telemetry related errors, we can link to this issue https://github.com/nginxinc/nginx-gateway-fabric/issues/1738